.versionrc.json

```diff
@@ -0,0 +1,9 @@
+{
+  "commitAll": true,
+  "bumpFiles": ["package.json", "selenium/package.json"],
+  "skip": {
+    "commit": true,
+    "tag": true,
+    "changelog": true
+  }
```

Comment:
I don't think we need any of this. We can get rid of the "Create release commit" step and just let `standard-version` do it all.

Reply:
We'd need to add an ["updater" script](https://github.com/conventional-changelog/standard-version#bumpfiles-packagefiles-and-updaters) for bumping the version in `pom.xml` btw.